File: Projetos/ACBrLib/Android/NFe/ACBrLibNFe/src/main/java/br/com/acbr/lib/nfe/notafiscal/IBSCBSProdutoNFe.java
package br.com.acbr.lib.nfe.notafiscal;

import br.com.acbr.lib.comum.dfe.CSTIBSCBS;
import br.com.acbr.lib.comum.dfe.cClassTribIBSCBS;
import br.com.acbr.lib.comum.dfe.gIBSCBS;
import br.com.acbr.lib.nfe.gIBSCBSMono;
import br.com.acbr.lib.nfe.gTransfCred;

public class IBSCBSProdutoNFe {

    public IBSCBSProdutoNFe(){
        gIBSCBS = new gIBSCBS();
        gIBSCBSMono = new gIBSCBSMono();
        gTransfCred = new gTransfCred();
    }

    private CSTIBSCBS CST;
    private cClassTribIBSCBS cClassTrib;
    private gIBSCBS gIBSCBS;
    private gIBSCBSMono gIBSCBSMono;
    private gTransfCred gTransfCred;

    public CSTIBSCBS getCST() {
        return CST;
    }

    public void setCST(CSTIBSCBS CST) {
        this.CST = CST;
    }

    public cClassTribIBSCBS getcClassTrib() {
        return cClassTrib;
    }

    public void setcClassTrib(cClassTribIBSCBS cClassTrib) {
        this.cClassTrib = cClassTrib;
    }

    public gIBSCBS getgIBSCBS() {
        return gIBSCBS;
    }

    public void setgIBSCBS(gIBSCBS gIBSCBS) {
        this.gIBSCBS = gIBSCBS;
    }

    public gIBSCBSMono getgIBSCBSMono() {
        return gIBSCBSMono;
    }

    public void setgIBSCBSMono(gIBSCBSMono gIBSCBSMono) {
        this.gIBSCBSMono = gIBSCBSMono;
    }

    public gTransfCred getgTransfCred() {
        return gTransfCred;
    }

    public void setgTransfCred(gTransfCred gTransfCred) {
        this.gTransfCred = gTransfCred;
    }
}

File: Projetos/ACBrLib/Demos/C#/NFe/Imports/Dinamico/Shared/NotaFiscal/IBSCBSProdutoNFe.cs
using ACBrLib.Core.DFe;

namespace ACBrLib.NFe
{
    /// <summary>
    /// Imposto de Importação
    /// </summary>
    public class IBSCBSProdutoNFe
    {
        #region Constructors
        public IBSCBSProdutoNFe()
        {
            gIBSCBS = new gIBSCBS();
            gIBSCBSMono = new gIBSCBSMono();
            gTransfCred = new gTransfCred();
            gCredPresIBSZFM = new gCredPresIBSZFM();

        }
        #endregion Constructors
        #region Properties

        /// <summary>
        /// Código de Classificação Tributária do IBS e CBS
        /// </summary>
        public CSTIBSCBS CST { get; set; }

        /// <summary>
        /// Código de Classificação Tributária do IBS e CBS
        /// </summary>
        public cClassTribIBSCBS cClassTrib { get; set; }  
        
        /// <summary>
        /// Grupo de Informações do IBS e da CBS
        /// </summary>
        public gIBSCBS gIBSCBS { get; }

        public gIBSCBSMono gIBSCBSMono { get; }

        public gTransfCred gTransfCred { get; }      
        
        public gCredPresIBSZFM gCredPresIBSZFM { get; }

        #endregion Properties
    }
}